infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerExposer.java

```diff
@@ -114,6 +102,21 @@
  *               servicePort: [8080|web-app]  ---->> Service.spec.ports[0].[port|name]
  * </pre>
  *
+ * <pre>
+ *   Host-Based Ingress exposing service's port:
```

Comment:
Maybe it worth to move examples to different strategies and add `{@see StrategyInterfaceClass}` link to the class? WDYT @sleshchenko @guydaichs ?

Reply:
@garagatyi I think it would be better to move examples to different strategies.